### DISCUSSION TITLE
Does Tailwind CSS support selector like regular CSS?

### DISCUSSION THREAD

**Discussion by Mubeen-Ul-Hassan** [tailwindlabs/tailwindcss#13893] [Help] [answered]
I was applying style to lists, and applying style to each element in Tailwind is frustrating. Is there a way to avoid this and apply a style to all list elements at once, similar to how we can do it in regular CSS by selecting the 'li' element?

**Comment by wongjn** [comment, marked as answer]
You could use an [arbitrary variant](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-variants) like:

```html
<div class="[&_li]:bg-red-500">
  <ul>
    <li>…</li>
    <li>…</li>
```
```html
<ul class="[&_li]:bg-red-500">
  <li>…</li>
  <li>…</li>
```
```html
<ul class="[&>li]:bg-red-500">
  <li>…</li>
  <li>…</li>
```
Or perhaps, using the [direct children variant](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-direct-children):
```html
<ul class="*:bg-red-500">
  <li>…</li>
  <li>…</li>
```
Live examples in [this Tailwind Play](https://play.tailwindcss.com/7ZXKpgFhj2).

**Reply by Mubeen-Ul-Hassan** [reply]
Thank you @wongjn
